pwiz_tools/Skyline/Controls/Graphs/GraphFullScan.cs

```diff
@@ -766,12 +766,16 @@ private void  CreateSingleScan(out double[] massErrors)
                 selectionMatch = ReferenceEquals(selection.NodeTranGroup?.Id, (dataPrecursor?.Id as Transition)?.Group);
             }
 
-            var currentTransition =
-                _msDataFileScanHelper.ScanProvider.Transitions[_msDataFileScanHelper.TransitionIndex];
-
             if (_showIonSeriesAnnotations && _msDataFileScanHelper.Source == ChromSource.fragment)
             {
-                var nodePath = DocNodePath.GetNodePath(currentTransition.Id, _documentContainer.DocumentUI);
+                DocNodePath nodePath = null;
+                if (_msDataFileScanHelper.ScanProvider.Transitions.Length > _msDataFileScanHelper.TransitionIndex - 1)

```

Comment:
Why is this "-1" there? That looks incorrect.

Reply:
You are right, of course. :-) Sorry, that was a brain fart.